lib/src/navigation_destination.dart

```diff
@@ -0,0 +1,8 @@
+// Groups together the path and arguments of a navigation destination
```

Comment:
Can you please make this a `///` comment just above the class?

Reply:
Thanks, this was just a mistake on my part.